cfy_manager/constants.py

```diff
@@ -39,9 +39,10 @@
 MANAGER_RESOURCES_HOME = join(REST_HOME_DIR, 'resources')
 AGENT_ARCHIVES_PATH = '{0}/packages/agents'.format(MANAGER_RESOURCES_HOME)
 
-USER_CONFIG_PATH = join(CLOUDIFY_HOME_DIR, 'config.yaml')
+CONFIG_FILE_NAME = 'config.yaml'
+USER_CONFIG_PATH = join(CLOUDIFY_HOME_DIR, CONFIG_FILE_NAME)
```

Comment:
Is this still used anywhere? It seems like something that's likely to lead to bugs if anywhere uses it rather than receiving the config that is loaded by the new methods?

Reply:
Yup, it is.  The installation process modifies the default configuration (/etc/cloudify/config.yaml) so it is still used in functions like _own_config_file or dump_config.

Reply:
fixed in 534f938d